collection/webcam/capture-webcam-image.yml

```diff
@@ -0,0 +1,28 @@
+rule:
+  meta:
+    name: capture webcam image
+    namespace: collection/webcam
+    author: johnk3r
+    scope: function
+    att&ck:
+      - Collection::Video Capture [T1125]
+    examples:
+      - a30101595f6f28ab2f4b0b2cd177c3c4d2ab34a355ab7761a3795d0887c24ada:0x4011C0
+  features:
+    - or:
+      - and:
+        - api: capCreateCaptureWindow
+        - basic block:
+          - and:
+            - api: SendMessage
+            - number: 0x40a = WM_CAP_DRIVER_CONNECT
+        - basic block:
+          - and:
+            - api: SendMessage
+            - number: 0x40B = WM_CAP_DRIVER_DISCONNECT
+        - optional:
+          - api: capGetDriverDescription
+      - basic block:
+        - and:
+          - api: SendMessage
+          - number: 0x419 = WM_CAP_FILE_SAVEDIB
```

Comment:
is this block indented correctly? alone it seems to be not as reliable?

Reply:
So, for the rule sample, if I leave this "block" in the main one, detection doesn't occur. In the screenshot I sent, you can see that this "block" is treated separately.

Note below:
  1º Only the constants and api were added. The conditioner used is an AND:
![ReverseMalw_w10-2021-06-10-10-56-30](https://user-images.githubusercontent.com/6247648/121538749-67b70300-c9db-11eb-94ed-92d53369a9bd.png)

  2º Only the constants and api were added. The conditioner used is an OR:
![ReverseMalw_w10-2021-06-10-10-55-10](https://user-images.githubusercontent.com/6247648/121538771-6c7bb700-c9db-11eb-867e-af6da3a2eb2d.png)


I didn't understand why this is and how I can't use the CAPA plugin in IDA, I'm not getting a better logic for it. If you have any tips, I will be grateful!

Reply:
My concern is if the
```yaml
      - basic block:
        - and:
          - api: SendMessage
          - number: 0x419 = WM_CAP_FILE_SAVEDIB
```
alone is enough to match on the capability

They are used in two separate functions so case 1 above doesn't match.